jetty-server/src/test/java/org/eclipse/jetty/server/handler/BadRequestLogHandlerTest.java

```diff
@@ -118,13 +117,8 @@ public void testLogHandler(String requestHeader, String expectedLog) throws Exce
         server.setConnectors(new Connector[] { connector });
 
         CaptureLog captureLog = new CaptureLog();
-
-        RequestLogHandler requestLog = new RequestLogHandler();
-        requestLog.setRequestLog(captureLog);
-        
-        requestLog.setHandler(new HelloHandler());
-
-        server.setHandler(requestLog);
+        server.setRequestLog(captureLog);
```

Comment:
Seems like we are stripping out all usages of `RequestLogHandler` in our test cases??

Reply:
Is the plan to remove it entirely?
If so, how does someone have unique RequestLog's per deployed WebApp?